src/webextension/list/components/item-summary.css

```diff
@@ -5,15 +5,38 @@
 .item-summary {
   border-bottom: 1px solid #d7d7db;
   padding: .5em;
+  font-size: 17px;
```

Comment:
This looks a bit on the large size in the management UI, but my main concern here is that it looks much *more* out of place in the doorhanger. It'll also make it harder for people to see long-ish titles in the doorhanger...